separated DOM managing from math functions

GonzoVM opened the pull request:
I've separated DOM managing from math functions adding a mid function 

```javascript
function changeNumber(conversionType)
``` 
that receive the id of the pushed button, then calls the isValid() function and, if is a correct input, calls the displayOutput() function giving it the correct conversion function as parameter.

joaquimgamero commented on public/index.html:
```diff
@@ -33,19 +33,22 @@ <h1>Base<sub>x</sub> conversor</h1>
-            value="Decimal"
-            onclick="bin2Dec(document.getElementById('num').value)"
+            value="decimal"
+            onclick="changeNumber(document.getElementById('bin2Dec').id)"
```
Every time you call this function: `changeNumber(...)`, why do you make the template get the value of the input? Why not doing:

`onclick="changeNumber('bin2Dec')"` 
`onclick="changeNumber('bin2Hex')"` 
`onclick="changeNumber('bin2Oct')"`

joaquimgamero replied:
In the end this is what arrives to the `changeNumber` function and anyway you know this value beforehand when you're defining the `onclick` listener, there's no reason to grab a value from the DOM for this.

GonzoVM replied:
I made it this way I wanted to use the id of the html, but yes, I think is a better way just calling the function with the needed value.

joaquimgamero replied:
What do you think are the benefits of using the ID of the input as a parameter?

GonzoVM replied:
none, it was there because I was using the value tag before and only changed the tag to id. I corrected that
https://github.com/GonzoVM/bin2dec/commit/3574b7f43e64689a41aa2fe0d3adce17e2910126

joaquimgamero replied:
good job, it looks better now!

joaquimgamero commented on src/calculator_options.js:
This looks good enough to be merged before we work on some other final improvements before calling it a good v1. If you can get rid of the commented code and do a general cleanup we can move this forward!

GonzoVM replied:
Done: https://github.com/GonzoVM/bin2dec/commit/58cbae4023611f25554ceb1fdc280c17eb5c245b

- I gave cleaned format to the html (I don't like how it shows it the prettifier addon, I think is easier to understand at this way.
- I added a bit of css to the output to distinguish it better.
- erassed old comments on js files.

joaquimgamero commented on src/calculator_options.js:
```diff
@@ -50,82 +40,68 @@ function hex2Bin(num) {
   octal = parseInt(num, 2).toString(8);
 
-  return displayOutput(octal, "octal");
+  return octal; // displayOutput(octal, "octal");
```
```suggestion
  return octal;
```

GonzoVM replied:
done
https://github.com/GonzoVM/bin2dec/commit/45ccd063cca5e4981b1ee8b8094e8c5ef9549ac7

GonzoVM commented:
I corrected some name functions, changed all '2' to 'To' on conversion functions (bin2Dec -> binToDec and so on) and changed 'changeNumber' to 'handleConversion'.

https://github.com/GonzoVM/bin2dec/commit/b3a7dcfb54db1374a442a83a70c478eca70f4d38
